consensus/bor/rest.go

```diff
@@ -47,8 +47,14 @@ func NewHeimdallClient(urlString string) (*HeimdallClient, error) {
 	return h, nil
 }
 
-func (h *HeimdallClient) FetchStateSyncEvents(fromID uint64, to int64) ([]*EventRecordWithTime, error) {
-	eventRecords := make([]*EventRecordWithTime, 0)
+func (h *HeimdallClient) FetchStateSyncEvents(fromID uint64, to int64, totalLimit int) ([]*EventRecordWithTime, error) {
+	var capacity int
+	if limit := totalLimit / 3; limit >= 1 {
```

Comment:
It may improve the readability by adding a comment here to briefly explain the purpose of `/ 3`, thanks!